### PR TITLE
Add gem devise

### PR DESCRIPTION
**Pull request by quynhthaivn**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
 
-gem 'rails', '~> 5.1.4'
-gem 'sqlite3'
-gem 'puma', '~> 3.7'
-gem 'sass-rails', '~> 5.0'
-gem 'uglifier', '>= 1.3.0'
-gem 'coffee-rails', '~> 4.2'
-gem 'jbuilder', '~> 2.5'
+gem "rails", "~> 5.1.4"
+gem "sqlite3"
```
Comment (NguyenHoangAnhDung) [inline Gemfile]: xóa lun sqlite cho rồi a =))

**Comment by vovanhai193** [comment]
OK

**Comment by NguyenHoangAnhDung** [comment]
OK
image

**Comment by Huyliver6793** [comment]
OK

File: Gemfile [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+gem "uglifier", ">= 1.3.0"
+gem "coffee-rails", "~> 4.2"
+gem "jbuilder", "~> 2.5"
+gem "devise"
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
-  gem 'capybara', '~> 2.13'
-  gem 'selenium-webdriver'
+  # debugger
```
Comment (vovanhai193) [inline Gemfile]: xóa

**Comment by quyenguyengoc** [comment]
687474703a2f2f77656264657369676e666c61742e686574656d6c2e6a702f77642d666c61742e636f6d2f6c74676d2f7370616e323031372d30322d32322e6a7067
OK.

**Pull request merged**